releng/org.eclipse.xtend.sdk.feature/p2.inf

```diff
@@ -1,2 +0,0 @@
-instructions.configure=org.eclipse.equinox.p2.touchpoint.eclipse.addRepository(location:http${#58}//download.eclipse.org/modeling/tmf/xtext/updates/composite/releases/,type:0,name:Xtext All In One - Releases,enabled:true); \
-  org.eclipse.equinox.p2.touchpoint.eclipse.addRepository(location:http${#58}//download.eclipse.org/modeling/tmf/xtext/updates/composite/releases/,type:1,name:Xtext All In One - Releases,enabled:true);
```

Comment:
Missing change in xtend.sdk.feature/build.properties?

Reply:
no does not contain: https://github.com/eclipse/xtext-umbrella/blob/master/releng/org.eclipse.xtend.sdk.feature/build.properties

Reply:
...